src/System.Memory/src/System/Buffers/ReadOnlySequence_helpers.cs

```diff
@@ -142,66 +142,66 @@ internal ReadOnlyMemory<T> GetFirstBuffer(in SequencePosition start, in Sequence
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal SequencePosition Seek(in SequencePosition start, in SequencePosition end, int count) => Seek(start, end, (long)count);
+        internal SequencePosition Seek(in SequencePosition start, in SequencePosition end, int offset) => Seek(start, end, (long)offset);
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal SequencePosition Seek(in SequencePosition start, in SequencePosition end, long count)
+        internal SequencePosition Seek(in SequencePosition start, in SequencePosition end, long offset)
         {
             GetTypeAndIndices(start.GetInteger(), end.GetInteger(), out SequenceType type, out int startIndex, out int endIndex);
 
             object startObject = start.GetObject();
-            object endObject = end.GetObject();
+            object endObject;
```

Comment:
Why was this change necessary?

Reply:
It will generate more optimized code.

Reply:
Also I renamed `count` to offset because it produce ArgumentException with wrong name for GetPosition methods.